release.js

```diff
@@ -28,7 +28,8 @@ const {version} = require('./package.json')
   await removeDir('dist')
 
   log('Switching to the dist branch...')
-  await execute('git checkout dist')
+  await execute('git checkout -B dist')
```

Comment:
create branch if needed: https://stackoverflow.com/a/26961416/1331425

Reply:
Hmm it seems like we should never recreate the `dist` branch because it is where all the previous "Release v${version}" commits live.. if it's recreated and force pushed to Github, we will wipe out all those previous releases, no?

Reply:
Ah but the next line overwrites the new local branch with what is on origin, right?

Reply:
Nevermind this line, it was my Git-newbie misunderstanding of how remote and local branches are working.